lib/sidebar-manager.js

```diff
@@ -414,5 +424,20 @@ exports.sidebarManager = {
                 this.showSidebar(storageManager.get('sidebarProps'));
             }
         }
+    },
+    loadNext: function() {
+        // get the time elapsed since the last cta completion
+        var timeElapsed = utils.getTimeElapsed(storageManager.get('lastSidebarCTACompleteTime'));
+
+        // get the amount we need to add to make up for the difference between CTA completion and when the timer should be done
+        var timeElapsedDiff = (intervals.defaultSidebarInterval - utils.getTimeElapsed(storageManager.get('lastSidebarCTACompleteTime')) > 0 ? intervals.defaultSidebarInterval - utils.getTimeElapsed(storageManager.get('lastSidebarCTACompleteTime')) : 0);
```

Comment:
Whoops, thats embarrassing, didn't even notice. I guess that is why we have code reviews :)